src/preact/base-element.js

```diff
@@ -475,13 +476,17 @@ export class PreactBaseElement extends AMP.BaseElement {
   }
 
   /**
+   * Register an action for AMP documents to execute an API handler.
+   *
+   * This has no effect on Bento documents, since they lack an Actions system.
+   * Instead, they should use `(await element.getApi()).action()`
    * @param {string} alias
    * @param {function(!API_TYPE, !../service/action-impl.ActionInvocation)} handler
    * @param {../action-constants.ActionTrust} minTrust
    * @protected
    */
   registerApiAction(alias, handler, minTrust = ActionTrust.DEFAULT) {
-    this.registerAction(
+    this.registerAction?.(
```

Comment:
This is good, this should operate fine in both Bento and AMP mode. In Bento, it won't actually register any of these action helpers, since the `BaseElement` we extend from shouldn't have it.